web_tree_image_tooltip/static/src/js/tooltip.js

```diff
@@ -0,0 +1,19 @@
+odoo.define('web_tree_image_tooltip.web_tree_image_tooltip',
+        function (require) {
+    "use strict";
+
+    var ListRenderer = require('web.ListRenderer');
+    ListRenderer.include({
+        events: _.extend({}, ListRenderer.prototype.events, {
+            'mouseover tbody tr td .o_field_image': '_onHoverRecord_img',
+        }),
+        _onHoverRecord_img: function (event) {
+            var img_src =
+                $(event.currentTarget).children('.img-fluid').attr('src')
+            $(event.currentTarget).tooltip({
+                title: "<img src="+img_src+" />",
+                delay: 0,
+            });
```

Comment:
Here can force 'show' to fix onmouseover issue
```suggestion
            }).tooltip('show');
```